MovieMVVM/MovieMVVM/Application/AppDelegate.swift

```diff
@@ -1,34 +1,61 @@
 // AppDelegate.swift
 // Copyright © RoadMap. All rights reserved.
```

Comment:
```suggestion
// Copyright ©. All rights reserved.
```
поменять везде на свое имя